pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -0,0 +1,377 @@
+"""Unit tests for the pylint checkers in
+:mod:`pylint.extensions.check_raise_docs`, in particular the raised exception
+documentation checker `ParamDocChecker`
+"""
+from __future__ import division, print_function, absolute_import
+
+import unittest
+import sys
+
+import astroid
+from astroid import test_utils
+from pylint.testutils import CheckerTestCase, Message, set_config
+
+from pylint.extensions.check_raise_docs import RaiseDocChecker
+
+
+class RaiseDocCheckerTest(CheckerTestCase):
+    """Tests for pylint_plugin.RaiseDocChecker"""
+    CHECKER_CLASS = RaiseDocChecker
+
+    def test_ignores_no_docstring(self):
+        node = test_utils.extract_node('''
+        def my_func(self):
+            raise RuntimeError('hi')
```

Comment:
You can annotate the code with ``#@`` for determining the node to be extracted. This becomes in this case:

```
raise_node = test_utils.extract_node('''
def my_func(self):
    raise RuntimeError('hi') #@
''')